packages/desktop-client/src/components/mobile/transactions/TransactionList.tsx

```diff
@@ -203,14 +203,13 @@ export function TransactionList({
               items={section.transactions.filter(
                 t => !isPreviewId(t.id) || !t.is_child,
               )}
-              addIdAndValue
             >
               {transaction => (
                 <TransactionListItem
                   key={transaction.id}
                   value={transaction}
-                  onPress={trans => onTransactionPress(trans)}
-                  onLongPress={trans => onTransactionPress(trans, true)}
+                  onPress={() => onTransactionPress(transaction)}
+                  onLongPress={() => onTransactionPress(transaction, true)}
```

Comment:
Is the `trans` object from the callback different that the `transaction`?

Reply:
I don't believe so, but I didn't test. The functionality is the same before and after

Reply:
Would it be fine to keep it as is?

Reply:
Ahh I fixed this on one of my PR: https://github.com/actualbudget/actual/pull/5875/files#diff-f558919867b6ae90e9d47cb380bf61308cc8f636a89c98d52d1fcb7738f6d605R74

Maybe the new linter is now able to detect overloads so we need to omit the default `onPress` from the `TransactionListItemProps`